extension/logstash/lib/logstash/outputs/doris.rb

```diff
@@ -131,22 +132,35 @@ def register
          end
       end
 
+      # Run named Timer as daemon thread
+      @timer = java.util.Timer.new("Doris Output #{self.params['id']}", true)
+      # The queue in Timer is unbounded and uncontrollable, so use a new queue to control the amount
+      @count_block_queue = java.util.concurrent.ArrayBlockingQueue.new(128)
```

Comment:
I think you should use LinkedBlockingQueue here since it's very offen to do insert and delete.